### ISSUE TITLE
issue #1:  "Uncaught TypeError: Cannot set property 'src' of undefined"

### ISSUE THREAD

**Issue by xuepgao** [justjim1220/MMM-MyVideoPlayer#6]
@justjim1220 

Thank you for providing us this fantastic module!

I have met several problems.

In most cases, this module works fine. The first issue is I met an error: **"Uncaught TypeError: Cannot set property 'src' of undefined"** sometime. The console error log is as following:
```
[23846:0502/140405.448088:INFO:CONSOLE(138)] "swap video button pressed = modules/MMM-MyVideoPlayer/videos/b.mp4", source: http://127.0.0.1:8080/modules/MMM-MyVideoPlayer//MMM-MyVideoPlayer.js (138)
[23846:0502/140405.474703:INFO:CONSOLE(139)] "Uncaught TypeError: Cannot set property 'src' of undefined", source: http://127.0.0.1:8080/modules/MMM-MyVideoPlayer//MMM-MyVideoPlayer.js (139)
2019-05-02T14:04:05 <log> Browser Error: {
   "isTrusted": true
} (/home/pi/MagicMirror/modules/MMM-Logging/node_helper.js:37 Class.socketNotificationReceived)
```
After this error appeared, the player bar did not responded to click. For example, before the error appeared, the player is playing a video. I clicked another thumbnail, but the player still continuous play previous video. it did not respond to click.

Could you give me some suggestion? Thanks in advance.

**Comment by justjim1220** [comment]
I will take a look at this later today.

Apologies for all the issues. I don't use Pi, so I am grateful to have someone trying it and giving feedback on issues needing fixed.

**Comment by xuepgao** [comment]
Thanks for your reply.
Please let me know if you need me to work with you remotely to reproduce the problem.

**Comment by justjim1220** [comment]
I made a few changes to the js file fixing a few syntax errors.
Hopefully this will fix the errors you are receiving

Please do a 'git pull && npm i' to update to the most recent version.

Thanks!

**Comment by xuepgao** [comment]
@justjim1220 

Thanks for your efforts!
I upgraded the module to the latest version. Now I found there is only one situation left that will cause this failure. When you are playing video under full screen mode, you switched the page  via other module's notification, touch control module, etc.  Then you switched back to MMM-MyVideoPlayer page, exit full screen mode, and clicked another thumbnail, the error log appeared and the module did not respond to click.

OK. I already known the conditions under which this failure occurred. So if it is difficult to solve, let it be. It's no matter.
